src/lib/cometvisu-client-openhab.js

```diff
@@ -0,0 +1,112 @@
+/* cometvisu.js (c) 2010 by Christian Mayer [CometVisu at ChristianMayer dot de]
```

Comment:
It uses Server-side-events (SSE), which is basically a long-polling where the connection is not closed after data was send (so no reconnection overhead). Please take a look at: https://jersey.java.net/documentation/latest/sse.html. The new openHAB2 does not support long-polling anymore, so we need a new communication lib. And IMHO SSE is better than long-polling. I can already see the performance benefit of gettnig rid of the reconnection overhead, although this is an early version (of course this only recognizable, when you have lots of updates in your visu).
Another advantage is, that the browser cares about lost connections and reconnecting.
So if you want to unify the communication I would rather suggest to update the default backend to use SSE as well.

I think the code is much cleaner this way, than putting it all in one lib.